### DISCUSSION TITLE
Xbox series s controller configuration in Blueretro

### DISCUSSION THREAD

**Discussion by Predatorx21** [darthcloud/BlueRetro#302] [Q&A]
Hello DarthCloud 

I recently wanted to use an xbox series s controller in blueretro but I couldn't link it. 

What option should I put in the default and advanced settings for it to work??

@darthcloud

**Comment by darthcloud** [comment]
No config is needed to connect a controller:
 https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#xbox-one-s-series-xs--adaptive-controller

Update your Xbox controller firmware to latest via the [Xbox accessories Win10 app](https://www.microsoft.com/en-us/p/xbox-accessories/9nblggh30xj3).

**Reply by Predatorx21** [reply]
Hi DarthCloud

I need to know if it is possible to save two joystick configurations in Blueretro (one for the FPS games and one for the other games)
*I am using an Xbox series S joystick on Dreamcast.

@darthcloud
 I wait your answer. Thank you

**Reply by darthcloud** [reply]
Only one config at a time.
